app/assets/stylesheets/content/_forms.sass

```diff
@@ -535,6 +535,11 @@ fieldset.form--fieldset
   & > [data-tooltip], & > [class^="tooltip--"]
     padding-top: 0.6rem
 
+.form--field
+  & .-break-words > .form--label
+    text-overflow: ellipsis
+    overflow: hidden
```

Comment:
not necessarily - i do want the modificator here - not all usages of `form--label` should rely on `ellipsis` - this could potentailly lead to situations with checkboxes being truncated too much (e.g. in the case of settings, where we actually fixed this in the past.